assets/stylesheets/style.css

```diff
@@ -0,0 +1,580 @@
+body {
+  background-color: #f4f5f6;
+}
+
+.page-header,
+.main,
+.footer {
```

Comment:
I suggest you give these classes a ```width:100%```. To solve the horizontal scroll we get on tablet and mobile screen sizes